deploy/sre-prometheus/100-ocm-agent-operator.PrometheusRule.yaml

```diff
@@ -38,3 +38,15 @@ spec:
         link: "https://github.com/openshift/ops-sop/blob/master/v4/alerts/OCMAgentResponseFailureServiceLogsSRE.md#verify-cluster-pull-secrets"
       annotations:
         message: OCM Agent pull secret auth token is not valid.
+
+
+    # Alert SRE if the OCM agent service log metric has been sent more than 10 within a 3-day period.
+    - alert: OCMAgentServiceLogsSentExceedingLimit
+      expr: sum(increase(ocm_agent_service_log_sent_total[3d])) > 10
+      for: 1m
```

Comment:
We don't really need the `for`, might as well drop it instead of having it at 1 minute.

Reply:
Well the `for` keyword is used to avoid alert flapiness.
That said the `ocm_agent_service_log_sent_total` metric is not supposed to increase frequently.
So indeed we could have that removed.